Studio-Lovelies/GG-JointJustice-Unity
# Minor improvements to the demo case

## Summary
Adds the latest version of the .ink script from the Google Drive and adds three placeholders for the `Office`, `Kitchen` and `Reception` scenes.

Also removes unused code in f91b133.

## Testing instructions
1. Play `2-1-FullScene` in the Unity editor
2. Notice you can play two talk segments and navigate across `Office`, `Kitchen` and `Reception`

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  also includes the changes in #346, which
